### ISSUE TITLE
Failed to detect nodes in ipython/jupyter since v1.1.0

### ISSUE THREAD

**Issue by pwwang** [alexmojaki/executing#58]
```python console
Python 3.9.5 (default, Jun  4 2021, 12:28:51) 
Type 'copyright', 'credits' or 'license' for more information
IPython 8.1.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import sys 
   ...: from executing import Source, __version__ 
   ...:  
   ...:  
   ...: def get_node(): 
   ...:     frame = sys._getframe(1) 
   ...:     return Source.executing(frame).node 
   ...:  
   ...:  
   ...: node = get_node() 
   ...: print(node) 
   ...:
None

In [2]: __version__
Out[2]: '1.1.0'
```

Used to work with v1.0.0

```python console
Python 3.9.5 (default, Jun  4 2021, 12:28:51) 
Type 'copyright', 'credits' or 'license' for more information
IPython 8.1.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import sys 
   ...: from executing import Source, __version__ 
   ...:  
   ...:  
   ...: def get_node(): 
   ...:     frame = sys._getframe(1) 
   ...:     return Source.executing(frame).node 
   ...:  
   ...:  
   ...: node = get_node() 
   ...: print(node) 
   ...:
<ast.Call object at 0x7fb7de6748b0>

In [2]: __version__
Out[2]: '1.0.0'
```

**Comment by pwwang** [comment]
Related:  https://github.com/pwwang/datar/issues/149

**Comment by alexmojaki** [comment]
Released fix in 1.1.1
